app/process/watcher/prometheus/watcher.go

```diff
@@ -300,6 +280,10 @@ func (pw Watcher) prepareAndSetAssetMetric(networkId uint64,
 	bridgeAccount *model.AccountsResponse,
 	isNative bool,
 ) {
+	if !pw.prometheusService.GetIsMonitoringEnabled() {
```

Comment:
There is the same check in the setMetrics function. Do we need it one more time since prepareAndSetAssetMetric is called in setAssetsMetrics > setMetrics?